src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -2576,9 +2577,9 @@ export class PostgresStorageAdapter implements StorageAdapter {
     fieldNames: string[],
     indexName: ?string,
     caseInsensitive: boolean = false,
-    conn: ?any = null
```

Comment:
Also maybe a breaking change?

Reply:
Unlikely. The `conn` parameter is not in use anywhere, it seems to be obsolete. All tests pass and if we are missing a test case, then any break would reveal that for the better.